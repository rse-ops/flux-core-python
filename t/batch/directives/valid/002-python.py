#!/usr/bin/env python3
# Directives embedded in python docstring
def main():
    """
    flux: -N4
    flux: --queue=batch
    flux: --job-name="my python job"

    # Set some arbitrary user data:
    flux: --setattr=user.data='''
    flux: x, y, z
    flux: a, b, c
    flux: '''
    """
    run()


if __name__ == "__main__":
    main()
