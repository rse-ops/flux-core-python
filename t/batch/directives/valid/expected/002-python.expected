SETARGS(['-N4'])
SETARGS(['--queue=batch'])
SETARGS(['--job-name=my python job'])
SETARGS(['--setattr=user.data=x, y, z\na, b, c\n'])
